streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionGraph.java

```diff
@@ -80,6 +86,14 @@ public long getBuildTime() {
     return buildTime;
   }
 
+  public int incLastExecutionVertexIndex() {
```

Comment:
I think it's not necessary to emphasize last vertex, just `generateVertextId` is enough.

Reply:
Use 'generateExecutionVertexId' instead.